### PR TITLE
Add TI DAC[5-7]578 DACs

### PR DESCRIPTION
**Pull request by chschlue**
* Add TI DAC[5-7]578 octal DACs in TSSOP and QFN packages

DS http://www.ti.com/lit/ds/symlink/dac5578.pdf

DAC5578xPW
DAC5578xRGE

---

All contributions to the kicad library must follow the [KiCad library convention](http://kicad-pcb.org/libraries/klc/)

Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [x] Provide a URL to a datasheet for the symbol(s) you are contributing
- [x] An example screenshot image is very helpful
- [x] Ensure that the associated footprints match the [official footprint library](https://github.com/kicad/kicad-footprints)
  - A new fitting footprint must be submitted if the library does not yet contain one.
- [x] If there are matching footprint PRs, provide link(s) as appropriate
- [x] Check the output of the Travis automated check scripts - fix any errors as required
- [x] Give a reason behind any intentional library convention rule violation.

### CODE REVIEW

**Comment by cpresser** [comment]
14-Pin TSSOP
- [x] ~~`!LDAC` should be an input~~ Nevermind, the screenshot is not up to date.
- [x] Description and Keywords are nice, including aliases
- [x] Pins numbering, electrical-type and position are good
- [x] Package and FP-Filters are good

QFN-24
- [x] Description and Keywords are nice, including aliases
- [ ] The EP should not be stacked, it is just recommended to connect it to GND.
- [x] Pins numbering, electrical-type and position are good
- [x] Package and FP-Filters are good

Question:
* The wildcard in the part name does mask the `S`. Is that the usual position for temperature variant?

**Comment by chschlue** [comment]
I'm not quite sure about the `S`.
The part is definitely called DAC[5-7]578 and the package is definitely {PW,RGE}.
These only exist with an infix `S` (as of now), but the meaning is not explained.
Other TI DACs use `M` and `Z` in its place, for example, meaning POR-to-midscale or zero output, respectively.

TL;DR: I wildcarded the `S` because it seems to be non-functional in the KLC sense. Do you want it back?

**Comment by cpresser** [comment]
I can also offer the `I`infix:
image

And one without an infix:
image

Judging from the two screenshots it is unlikely that its a temperature variant.
Having Mid-level or Low-level initial configuration is a functional thing.

I would suggest to not wildcard it since we don't really know what it is.

**Comment by chschlue** [comment]
I looked for a rationale for having DAC8750xRHA in the lib and found out it was actually your own addition :-)

That being said, I am also not convinced the S indicates temp rating.

@evanshultz requested the wildcard for #960, maybe he can shed some light or point to an actual naming convention document?

**Comment by cpresser** [comment]
@evanshultz ping? Any hints on the wildcard stuff?

**Comment by chschlue** [comment]
@evanshultz ping

**Comment by cpresser** [comment]
I looked a little bit more and found some documents that have info about TI part numbers.
http://www.ti.com/lit/ml/slyb191/slyb191.pdf
http://www.ti.com/lit/an/sboa142a/sboa142a.pdf

Unfortunately none of them have a nomenclature that matches this DAC or comes close.

I am going to merge this now with the wildcard in the name. Most likely the wildcard is not wrong. And we don't have any source of information to clarify.
